crates/net/network/src/transactions.rs

```diff
@@ -404,6 +408,7 @@ where
     fn on_command(&mut self, cmd: TransactionsCommand) {
         match cmd {
             TransactionsCommand::PropagateHash(hash) => self.on_new_transactions(vec![hash]),
+            TransactionsCommand::PropagateHashTo(hash, peer ) => self.on_new_transactions(vec![hash]),
```

Comment:
since we want to send the hashes to the given peer, we need a new function that basically does this:

https://github.com/paradigmxyz/reth/blob/11ee5d75308d4a7519fdecba4a0fa5f51c61cf0e/crates/net/network/src/transactions.rs#L292-L292

we also need to check we have the peer and insert the transactions into the `Peer`'s cache

Reply:
Thank you so much, i'm working on it.


Please can you tell me the time i will have for this issue ? I'm doing my best to understand what needed and work on it, i hope i can get a time window 

Thank you again and i'm sorry for my basics questions

Reply:
take your time there's no deadline for this :)

I'm happy to answer questions to help get this over the line